homeassistant/components/eq3btsmart/__init__.py

```diff
@@ -53,42 +57,39 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         ble_device=device,
     )
 
-    eq3_config_entry = Eq3ConfigEntryData(eq3_config=eq3_config, thermostat=thermostat)
-    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = eq3_config_entry
-
+    entry.runtime_data = Eq3ConfigEntryData(
```

Comment:
since this is affecting other entities and we like to keep PRs as small as possible: Better to do that in a prereq PR